bin/serve.hs

```diff
@@ -11,8 +12,8 @@ import Crypto.JOSE.JWK (fromKeyMaterial)
 import Crypto.JOSE.JWA.JWK (KeyMaterial(OctKeyMaterial), OctKeyParameters(..))
 import Crypto.JOSE.Types (Base64Octets(..))
 
-import qualified Prototype.Database as Database (newHandle)
-import Prototype.Server (api, server)
+import qualified "start-servant" Prototype.Database as Database (newHandle)
```

Comment:
What's the rational for using `-XPackageImports` ? I thought it was a nice thing to provide drop-in replacement library, or somehow "parameterize" an application, but I don't understand the case for it here.

By the way, this breaks the `serve.sh` script. I guess that the library has to be compiled beforehand ? Is it still possible to use `runghc` or does it means only `cabal repl` will work ?

This also breaks `ghcid.sh`.

Reply:
Yes, we can remove it. I was mostly adding them here so that I could explicitly see how Cabal was resolving these modules: through the `executable` or through the `library`. They can turn in handy when we have conflicting module names supplied through different dependencies, or for documentation. But I don't have a strong opinion on this, we can do without them.